internal/model/sharedpullerstate.go

```diff
@@ -250,7 +250,7 @@ func (s *sharedPullerState) Progress() *pullerProgress {
 		CopiedFromElsewhere: s.copyTotal - s.copyNeeded - s.copyOrigin,
 		Pulled:              s.pullTotal - s.pullNeeded,
 		Pulling:             s.pullNeeded,
-		BytesTotal:          files.BlocksToSize(total),
-		BytesDone:           files.BlocksToSize(done),
+		BytesTotal:          db.BlocksToSize(total),
```

Comment:
Mmm... It's not really a protocol thing, ideally it should go away and we should have a real size field stored away. I'm going to fix that when I continue refactoring this database stuff...